test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -11,6 +11,26 @@ Describe "New-PSSession basic test" -Tag @("CI") {
     }
 }
 
+Describe "Basic Auth over HTTP not allowed on Unix" -Tag @("CI") {
+    It "New-PSSession should throw when specifying Basic Auth over HTTP on Unix" -skip:($IsWindows) {
+        $password = ConvertTo-SecureString -String "password" -AsPlainText -Force
+        $credential = [PSCredential]::new('username', $password)
+        try
+        {
+            New-PSSession -ComputerName 'localhost' -Credential $credential -Authentication Basic
+            throw "New-PSSession should throw"
+        }
+        catch
+         {
+            $_.Exception | Should -BeOfType [System.Management.Automation.Remoting.PSRemotingTransportException]
```

Comment:
We should use 
```c#
$ex = { ... } | Should -Throw -PassThrou "FullyQualifiedErrorId"
$ex.Exception ...
```

Reply:
Example: https://github.com/PowerShell/PowerShell/blob/986a9eeecdb800cde928a0d9e7e926fe5086d4cf/test/powershell/Modules/Microsoft.PowerShell.Utility/Remove-Variable.Tests.ps1#L11 

Reply:
Fixed